src/location/location.service.ts

```diff
@@ -11,6 +12,28 @@ export class LocationService {
     return locations;
   }
 
+  async createLocation(data: CreateLocationDto) {
+    try {
+      await this.prismaService.location.create({
+        data,
+      });
+
+      return { message: 'Successfully created a location' };
+    } catch (error) {
+      if (error.code === 'P2002') {
```

Comment:
Noticed error.code === "P2002" is being used often and I had to search pa kung ano ang P2002 na error. How about we store constants of all of the error codes? So here sa conditions we can do:
`if(error.code === UNIQUE_CONSTRAINT_ERROR)`  Makes it a lot more readable.